python/pyspark/pandas/series.py

```diff
@@ -3561,6 +3563,8 @@ def rank(self, method: str = "average", ascending: bool = True) -> "Series":
             * dense: like 'min', but rank always increases by 1 between groups
         ascending : boolean, default True
             False for ranks by high (1) to low (N)
+        numeric_only : bool, optional
+            Rank only numeric columns if set to True.
```

Comment:
and how about refine this description something like: "For Series objects, it returns empty Series if set to True when Series is not numeric type ." ?

I think pandas document for Series doesn't look like proper enough.

Reply:
Rephrased. Let me know if you still think it is unclear. Thanks